napalm/junos.py

```diff
@@ -100,7 +103,7 @@ def get_facts(self):
             'os_version': output['version'],
             'hostname': output['hostname'],
             'fqdn': output['fqdn'],
-            'uptime': uptime,
+            'uptime': string_parsers.convert_uptime_string_seconds(uptime),
```

Comment:
Never mind, I was thinking about interfaces, but this is about facts and those could use the translation indeed